### ISSUE TITLE
Please provide windows installers

### ISSUE THREAD

**Issue by Alxandr** [princejwesley/Mancy#17]
There is a npm package for creating windows installers out of electron projects IIRC. Would be really nice to have an installer.

**Comment by senthilporunan** [comment]
We are having windows supported version for both 32 bit and 64 bit. You can download it from here. 

https://github.com/princejwesley/Mancy/releases

**Comment by Alxandr** [comment]
Yes. I didn't say "windows supported versions". I said a "windows installer". You know, a exe/msi that you run, that unpacks all the files at all the right places, and create shortcuts where you want them etc.

**Comment by senthilporunan** [comment]
ok we will try to  add them in next version.

**Comment by Alxandr** [comment]
In case you didn't know, there is as said a npm package for making windows installers for electron apps: https://www.npmjs.com/package/electron-installer-squirrel-windows

**Comment by princejwesley** [comment]
@Alxandr For installers, we have to buy digitial certificates from reputed certificate authorities and renew it every year.

**Comment by Alxandr** [comment]
@princejwesley https://github.com/atom/grunt-electron-installer#sign-your-installer-or-else-bad-things-will-happen

Or, you know, you can just not sign it. Sure, signed would be nicer, cause it doesn't trigger smartscreen filter, but still, it's a free service you're providing, that I choose to install on my own risk. I don't expect it to be signed.

**Comment by nblackburn** [comment]
+1
